components/table/index.en-US.md

```diff
@@ -126,6 +126,7 @@ One of the Table `columns` prop for describing the table's columns, Column has t
 | dataIndex | Display field of the data record, could be set like `a.b.c`, `a[0].b.c[1]` | string | - |  |
 | defaultFilteredValue | Default filtered values | string\[] | - |  |
 | defaultSortOrder | Default order of sorted values | 'ascend' \| 'descend' | - |  |
+| editable | Whether the column is `editable` | boolean | `false` |  |
```

Comment:
We don't provide `editable` cell now.

Reply:
I got it from this example for table with editable rows https://ant.design/components/table/#components-table-demo-edit-row. If editable isn't defined, typescript's compiler outputs an error:

```
 Type '{ key: string; title: JSX.Element; dataIndex: string; editable: boolean; }' is not assignable to type 'ColumnProps<P>'.
````

Reply:
You have to extend ColumnProps in this case, rather than change it in antd's source code.